docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -201,6 +157,80 @@ TBD / WIP - some high level thoughts
 
 TODO: Add image / gif / video (maybe not for this one)
 
+## Data sources
+
+### Tempo data source: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute
+
+<!-- Joey Tawadrous, Jen Villa -->
+
+_Experimental in Grafana Open Source and Enterprise_
+
+(Requires Tempo or Grafana Enterprise Traces (GET) v2.2 or greater)
+
+We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Tempo's [metrics summary API](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/metrics-summary/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
+
+This feature is disabled by default. To enable it, use the `metricsSummary` [experimental feature toggle](https://grafana.com/docs/grafana/next/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles).
```

Comment:
@imatwawana -- note that my link has a "next" in there. I'm not sure that's correct.